test/cpp/naming/resolver_component_test.cc

```diff
@@ -438,10 +446,28 @@ void RunResolvesRelevantRecordsTest(void (*OnDoneLocked)(void* arg,
   GPR_ASSERT(gpr_asprintf(&whole_uri, "dns://%s/%s",
                           FLAGS_local_dns_server_address.c_str(),
                           FLAGS_target_name.c_str()));
+  gpr_log(GPR_DEBUG, "resolver_component_test: --enable_srv_queries: %s",
+          FLAGS_enable_srv_queries.c_str());
+  grpc_channel_args* resolver_args = nullptr;
+  // By default, SRV queries are disabled, so tests that expect no SRV query
+  // should avoid setting any channel arg. Test cases that do rely on the SRV
+  // query must explicitly enable SRV though.
+  if (FLAGS_enable_srv_queries == "True") {
+    grpc_arg srv_queries_arg;
```

Comment:
This can use `grpc_channel_arg_integer_create()`.

Reply:
Done. thanks for the review